### PR TITLE
[Chips] Pixel align subviews

### PR DESCRIPTION
**Pull request by samnm**
Padding for these samples has been set to 0,0,0,0 for testing. 

Before:
before

After:
after

### CODE REVIEW

File: components/private/Math/src/MDCMath.h [hunk 1]
```diff
@@ -172,6 +172,14 @@ static inline CGPoint MDCPointRoundWithScale(CGPoint point, CGFloat scale) {
   return CGPointMake(MDCRound(point.x * scale) / scale, MDCRound(point.y * scale) / scale);
 }
 
+static inline CGSize MDCSizeCeilWithScale(CGSize size, CGFloat scale) {
```
Comment (jverkoey) [inline components/private/Math/src/MDCMath.h]: Docs.

File: components/Chips/src/MDCChipView.m [hunk 2]
```diff
@@ -79,7 +79,10 @@
 
 static CGRect CGRectVerticallyCentered(CGRect rect, UIEdgeInsets padding, CGFloat height) {
   CGFloat viewHeight = CGRectGetHeight(rect) + padding.top + padding.bottom;
-  return CGRectOffset(rect, 0, (height - viewHeight) / 2);
+  CGFloat yValue = (height - viewHeight) / 2;
+  CGFloat scale = UIScreen.mainScreen.scale;
```
Comment (jverkoey) [inline components/Chips/src/MDCChipView.m]: Should this accept a scale?

**Review by jverkoey — DISMISSED** [review]
Result looks good - just left some small comments re: docs and consistency.

File: components/private/Math/src/MDCMath.h (hunk 1 above)
Comment (samnm) [reply components/private/Math/src/MDCMath.h]: Done.

File: components/Chips/src/MDCChipView.m (hunk 2 above)
Comment (samnm) [reply components/Chips/src/MDCChipView.m]: Done.

**Comment by samnm** [comment]
@jverkoey PTAL if you have a moment

**Review by jverkoey — APPROVED** [review]

**Pull request merged**